src/components/accounts/SetRecoveryInfo.js

```diff
@@ -86,22 +123,42 @@ class SetRecoveryInfo extends Component {
          ...this.state,
          isLegit: this.state.isLegit && !this.props.formLoader
       }
-      const { sentSms } = this.props
-      
-      return (
-         <AccountFormContainer 
-            title={sentSms ? `Enter your Code` : `Protect your Account`}
-            text={sentSms ? `We sent you a 6-digit code via SMS text. Please enter it below to find your account.` : `Enter your phone number to make your account easy for you to recover in the future.`}
-         >
-            <AccountFormSection handleSubmit={this.handleSubmit} requestStatus={this.props.requestStatus}>
-               <SetRecoveryInfoForm
-                  {...combinedState}
-                  handleChange={this.handleChange}
-                  skipRecoverySetup={this.skipRecoverySetup}
-               />
-            </AccountFormSection>
-         </AccountFormContainer>
-      )
+      const { sentSms } = this.props;
+      const { sentEmail } = this.state;
+
+      if (this.state.BackupWithEmail) {
+         return (
+            <AccountFormContainer 
+               title={sentEmail ? 'Confirm Recovery Setup' : 'Protect your Account'}
```

Comment:
I think it makes sense to rename `sentSms` to `sentMessage` and use it in both cases instead of 2 different fields to signal same status.

Reply:
Good point - will change